### DISCUSSION TITLE
[general] JOB: Short-term contract building {targets} pipelines

### DISCUSSION THREAD

**Discussion by noamross** [ropensci/targets#1199] [General]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

Hi all! I thought this might be a good place We're looking for a research software/data engineer for a short-term position building data integration and QA/QC pipelines for a field + lab research project.  We use {targets} extensively, along GitHub Actions to collect data via Web APIs of our various sources (AirTable, Google Sheets, and Open Data Kit).  If you have some experience in these things, we'd love to hear from you!

$50-60 per hour, ~12 weeks full-time, remote from anywhere.

More info here: https://bit.ly/eha-rse-ad

**Comment by wlandau** [comment]
Hi @noamross, have you had any luck with job candidates?

**Comment by noamross** [comment]
We have :) . I can close this.
